src/templates/layouts/pressRelease/index.tsx

```diff
@@ -0,0 +1,155 @@
+import { PressRelease as FormattedPressRelease } from '@/types/formatted/pressRelease'
+import { LovellStaticPropsResource } from '@/lib/drupal/lovell/types'
+import { LovellSwitcher } from '@/templates/components/lovellSwitcher'
+import { formatDate } from '@/lib/utils/helpers'
+import { recordEvent } from '@/lib/analytics/recordEvent'
+import { ContentFooter } from '@/templates/common/contentFooter'
+
+export const PressRelease = ({
+  title,
+  releaseDate,
+  pdfVersion,
+  introText,
+  address,
+  fullText,
+  contacts,
+  downloads,
+
+  /*listing,*/
+  lovellVariant,
+  lovellSwitchPath,
+}: LovellStaticPropsResource<FormattedPressRelease>) => {
+  return (
+    <>
+      <div>
+        <main className="va-l-detail-page va-facility-page">
+          <div className="usa-grid usa-grid-full">
+            {/* nav here */}
+            <div className="usa-width-three-fourths">
+              <article className="usa-content">
+                <LovellSwitcher
+                  currentVariant={lovellVariant}
+                  switchPath={lovellSwitchPath}
+                />
+                <section className="vads-u-margin-bottom--5">
+                  <h1 className="vads-u-margin-bottom--2p5">{title}</h1>
+                  <p className="vads-u-margin-bottom--0p5">PRESS RELEASE</p>
+                  <p className="vads-u-font-weight--bold vads-u-margin-bottom--3 vads-u-margin-top--0">
+                    {releaseDate || formatDate(releaseDate)}
+                  </p>
+                  <div>
+                    <button
+                      type="button"
+                      className="va-u-margin-right--4 va-button-link"
+                      onClick={() => {
+                        window.print()
+                        return false
+                      }}
+                    >
+                      <i className="fa fa-print vads--u-padding-right--1"></i>
+                      Print
+                    </button>
+                    {pdfVersion && (
+                      <a href={pdfVersion.field_document.uri.url} download>
```

Comment:
this should get handled in the formatter as well, `pdfVersion: entity.field_pdf_version.field_document.uri.url,`. this would make it `<a href={pdfVersion} download>`